backend/lucene/src/main/java/org/hibernate/search/backend/lucene/scope/model/impl/LuceneScopeModel.java

```diff
@@ -148,4 +149,15 @@ public void checkNestedField(String absoluteFieldPath) {
 			throw log.unknownFieldForSearch( absoluteFieldPath, getIndexesEventContext() );
 		}
 	}
+
+	public Set<String> getNestedDocumentPaths(String absoluteFieldPath) {
+		HashSet<String> nestedDocumentPaths = new HashSet<>();
```

Comment:
Why did you allow multiple document paths? That would mean that some targeted indexes have a different structure, and currently, for every projection, we throw an exception when the structure differs between indexes.

I'm really not sure it's a good idea to try and support that kind of things immediately. It would be safer to just throw an exception.
Whatever you choose... please add a test?

Reply:
Indeed I was not sure to allow that.
Let's raise an exception for.